appstore/developer_portal_api.py

```diff
@@ -292,7 +292,7 @@ def submit_new_release(app_id):
     uuid = appinfo['uuid']
     version = appinfo['versionLabel']
         
-    if uuid != app.app_uuid:
+    if str(uuid) != str(app.app_uuid):
```

Comment:
why?

Reply:
I started to suddenly get this on my machine:

```python
...
uuid = appinfo['uuid']

print(type(uuid))
print(type(app.app_uuid))
print(uuid == app.app_uuid)
```

Output:
```
<class 'str'>
<class 'uuid.UUID'>
False
```